src/plugins/lua/ratelimit.lua

```diff
@@ -560,14 +560,11 @@ if opts then
     -- new way of setting limits
     fun.each(function(t, lim)
       if type(lim) == 'table' then
-        if type(lim[2]) == 'number' then
-          settings[t] = lim
-        elseif type(lim[2]) == 'string' then
-          settings[t] = lim
-          local plim = parse_string_limit(lim[2])
-          if plim then
-            settings[t][2] = plim
-          end
+        settings[t] = lim
+      elseif type(lim) == 'string' then
+        local plim = parse_string_limit(lim)
+        if plim then
+          settings[t] = {0, plim, 1}
```

Comment:
If we have `some_ratelimit = "2 / 1m";` and "2" is the size of the bucket then what is leak rate? Is it supposed to be 1 per minute? Code seems to expect that this in fact part of the leak rate and not bucket size: which I understood we'd want to always be zero, rather than specified in configuration at all.